src/erlang_ls.erl

```diff
@@ -22,38 +23,100 @@ main(Args) ->
   lager:info("Started erlang_ls server", []),
   receive _ -> ok end.
 
+-spec print_version() -> ok.
+print_version() ->
+  {ok, Vsn} = application:get_key(erlang_ls, vsn),
```

Comment:
Maybe use `?APP` instead of `erlang_ls`.

Reply:
Squashed it into the main commit